app/views/admins/members/new.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= simple_form_for [:admin, @member] do |f| %>
+  <%= f.input :first_name, label: 'First name:', required: true %><br />
+  <%= f.input :last_name, label: 'Last name:', required: true %><br />
+  <%= f.input :email, label: 'Email:', required: true %><br />
+  <%= f.submit 'Create' %>
+<% end %>
```

Comment:
It looks like the new and edit forms are exactly the same. Could you pull the two out into a partial called `app/views/admins/members/_form.html.erb` and render that ?